acceptance-tests/runner/runtime-plugins/runtime-deps/pom.xml

```diff
@@ -69,14 +69,19 @@
         <dependency>
             <groupId>io.jenkins.blueocean</groupId>
             <artifactId>blueocean-executor-info</artifactId>
-            <version>[1.14.0-SNAPSHOT,)</version>
+            <version>[1.23.0-SNAPSHOT,)</version>
         </dependency>
 
         <!-- following are required to run on newer jenkins versions (after 2.7) as it changed how classloading works a bit for plugins -->
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>structs</artifactId>
-            <version>1.17</version>
+            <version>1.20</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>apache-httpcomponents-client-4-api</artifactId>
```

Comment:
`dependencyManagement`?

Reply:
would be better but this is only use for BO testing this dependency cannot be used by any project. So I have been lazy :)